viadot/sources/cloud_for_customers.py

```diff
@@ -0,0 +1,256 @@
+import re
+import requests
+import pandas as pd
+
+from copy import deepcopy
+from typing import Any, Dict, List, Optional
+from pydantic import BaseModel
+from urllib.parse import urljoin
+from viadot.exceptions import CredentialError
+
+from ..utils import handle_api_response
+from ..config import get_source_credentials
+from .base import Source
+
+
+class CloudForCustomersCredentials(BaseModel):
+    username: str  # CloudForCustomers username (e.g username@{tenant_name}.com).
+    password: str  # CloudForCustomers password.
```

Comment:
I don't this these comments are necessary, maybe besides the example which could just read "# eg. username@{tenant_name}.com"

Reply:
Done